smapply/api.py

```diff
@@ -206,17 +214,15 @@ def parse_webhook(webhook):
         webhook (WebhookRequestSMA): WebhookRequestSMA instance
 
     """
-    try:  # pylint:disable=too-many-nested-blocks
+    try:
         body_json = json.loads(smart_text(webhook.body))
         field_mapping = {'id': 'submission_id', 'award': 'award_id', 'user_id': 'user_id'}
-        required_fields = field_mapping.keys()
+        required_fields = ['id', 'user_id']
```

Comment:
I am referring to these fields here. Why this change?

Reply:
`field_mapping.keys()` = `['id', 'award', 'user_id']`

The field `id` is required in the webhook body, and maps to `submission_id` in the database. The required fields are applied to the webhook body. Requiring `id` is requiring `submission_id`.

This change does not remove the requirement for `submission_id`, it only removes the requirement for `award`/`award_id`.

Reply:
Thanks for the explanation! 

I think there is a misunderstanding about the meaning of award. Award_id is used for retrieval of program info in SMApply, and it corresponds to our klass/bootcamp. So I don't think there can be a submission without the award(program) that they are applying for.

Reply:
Just tested and you are right, there is an award.